compatibility/BUILD

```diff
@@ -18,6 +18,15 @@ platform_versions = [
     "1.0.1-snapshot.20200417.3908.1.722bac90",
 ]
 
+[
+    sh_binary(
```

Comment:
for better or for worse `sh_binary` seems like the easiest way to switch out the runfiles of a Haskell target without recompiling it.